django_netjsongraph/api/generics.py

```diff
@@ -25,6 +27,20 @@ class BaseNetworkGraphView(generics.RetrieveAPIView):
     serializer_class = NetworkGraphSerializer
 
 
+class BaseNetworkGraphHistoryView(APIView):
+    """
+    History of a specific topology returned
+    in NetJSON NetworkGraph format
+    """
+
+    def get(self, request, pk, format=None):
+        topology = get_object_or_404(self.topology_model, pk)
+        date = request.query_params.get('date')
```

Comment:
write a test in which you supply a rubbish date here, make it fail and then find a fix (great occasion to learn TDD)